decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
Strictly speaking the 100% equivalent behavior would be using `to_s`, that's why I'm surprised. 

```
[1] pry(main)> class A
[1] pry(main)*   def to_s
[1] pry(main)*     "hola"
[1] pry(main)*   end  
[1] pry(main)*   
[1] pry(main)*   def to_str
[1] pry(main)*     "hi"
[1] pry(main)*   end  
[1] pry(main)* end  
=> :to_str
[2] pry(main)> "#{A.new}"
=> "hola"
[3] pry(main)> 
```

Also note that if the result of `content_tag` is already a `String` instance, I'd say you don't need either.